app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Let's change the association between a user and a comment to suit the project requirement. The ERD shows `author_id` instead of `user_id`. We want a comment to belong to an `author`.
HINT: `belongs_to :author, class_name: 'User'`

Reply:
Thanks, done!